onboarding_documentation/ingestion/ingestion_tutorial.md

```diff
@@ -0,0 +1,218 @@
+
+    For this example, using the metadata manifest co-located with the data, we will use the following command:
+
+    `python metamist/parser/sample_file_map_parser.py --project sandbox --search-path gs://cpg-sandbox-test-upload/onboarding_fastqs/ gs://cpg-sandbox-test-upload/onboarding_fastqs/onboarding_fastq_sample_file_map.csv`
+
+    We will modify our command with the following flags:
+
+    - --dry-run (prepares the run without committing it.)
+    - --confirm (before updating the server, confirms with the user)
+    - --verbose (gives very detailed textual output surrounding your command)
+
+    `python metamist/parser/sample_file_map_parser.py --project sandbox --search-path gs://cpg-sandbox-test-upload/onboarding_fastqs/ --dry-run --confirm --verbose gs://cpg-sandbox-test-upload/onboarding_fastqs/onboarding_fastq_sample_file_map.csv`
+
+7. Run the parser script through your terminal
+
+    Use Visual Studio Code or your preferred IDE to perform this step and ensure that you have the Metamist repository opened as your project. Following instructions are provided for Visual Studio Code.
+
+    Open a new terminal by clicking the Terminal button in your menu bar and the ‘New Terminal’ option.
+
+    Due to the way that the paths have been defined in our command, ensure that you are in the correct directory first before execution. Ensure that you are at the ‘top level’ of the metamist repository by typing pwd in your terminal and hit enter. You should see a file path printed similar to the following:
+
+        /Users/{your name}/{storage location}...{/optional subfolders}/metamist
+
+    Paste the command composed in your terminal in step 6 and hit enter.
+
+    ⚠️ If you receive an error message that states that you have not configured your default project, please try one of the two following commands to set sandbox as your default project:
+    - `gcloud auth application-default set-quota-project cpg-sandbox`
+    - `gcloud config set project cpg-sandbox`
+
+8. Verify the outcome of your command in your terminal
+    If the  `--verbose` or `--confirm` flags have been provided, you will see descriptive text and a tabular representation in your terminal that summarises what has been ingested. Check that this corresponds to your metadata manifest file.
+
+    ![Descriptive text following successful parser run](./images/7_parser_output.png)
+
+    Read the descriptive text underneath the table to check which tables and ensure that you have the required number of inserts, again according to your metadata manifest file. We can see that there are 10 insertions per table, which corresponds to the 10 participants included in the original file.
+
+    {'participants': {'insert': 10, 'update': 0}, 'samples': {'insert': 10, 'update': 0}, 'sequencing_groups': {'insert': 10, 'update': 0}, 'assays': {'insert': 10, 'update': 0}, 'analyses': {'insert': 0}}
```

Comment:
The text here is a little discordant with the image, which shows 10 assays to _update_, not _insert_.